checkov/sca_package/scanner.py

```diff
@@ -5,116 +5,102 @@
 import time
 from collections.abc import Iterable, Sequence
 from pathlib import Path
-from typing import Tuple, Dict, Any
+from typing import Dict, Any
 
+import requests
 from aiomultiprocess import Pool
 
-from checkov.common.bridgecrew.platform_key import bridgecrew_dir
-from checkov.common.bridgecrew.vulnerability_scanning.integrations.package_scanning import package_scanning_integration
+from checkov.common.bridgecrew.platform_integration import bc_integration
+from checkov.common.util.file_utils import compress_file_gzip_base64, decompress_file_gzip_base64
 
-
-TWISTCLI_FILE_NAME = 'twistcli'
-CHECKOV_SEC_IN_WEEK = 604800
+SLEEP_DURATION = 2
+MAX_SLEEP_DURATION = 60
 
 
 class Scanner:
     def __init__(self) -> None:
-        self.twistcli_path = Path(bridgecrew_dir) / TWISTCLI_FILE_NAME
-
-    def setup_twictcli(self) -> None:
-        try:
-            if self.should_download():
-                if not os.path.exists(bridgecrew_dir):
-                    os.makedirs(bridgecrew_dir)
-                self.cleanup_twictcli()
-                package_scanning_integration.download_twistcli(self.twistcli_path)
-        except Exception:
-            logging.error("Failed to setup twictcli for package scanning", exc_info=True)
-            raise
```

Comment:
just to make sure, do we remove already what Noa implemented today?

Reply:
We spoke, it needs to be implemented on the sca_image runner - but on the sca_package runner it will become obsolete soon